frontend/src/components/common/image/image.tsx

```diff
@@ -9,15 +9,17 @@ type Props = {
   src: string;
   alt: string;
   isCircular?: boolean;
+  onClick?: () => void;
 };
 
-const Image: FC<Props> = ({ width, height, src, alt, isCircular }) => (
+const Image: FC<Props> = ({ width, height, src, alt, isCircular, onClick }) => (
```

Comment:
Should not your functional component specify return type?

Reply:
As far as I know, specifying `FC<Props>` should be enough

Reply:
> As far as I know, specifying FC<Props> should be enough
right